ranger/default.go

```diff
@@ -356,7 +356,7 @@ func defaultSessionStore(env trails.Environment, appName string) (session.Sessio
 
 // defaultServer constructs a default [*http.Server].
 func defaultServer(ctx context.Context) *http.Server {
-	host := trails.EnvVarOrString(hostEnvVar, DefaultHost)
+	host := trails.EnvVarOrString(hostEnvVar, "")
```

Comment:
Nit: use `os.Getenv(hostEnvVar)` since this returns the zero value if the env var is unset.